source/MaterialXView/Viewer.cpp

```diff
@@ -895,6 +889,57 @@ void Viewer::createAdvancedSettings(Widget* parent)
         _lightHandler->setEnvSampleCount(MIN_ENV_SAMPLE_COUNT * (int) std::pow(4, index));
     });
 
+    ng::Label* viewLabel = new ng::Label(advancedPopup, "Viewing Options");
```

Comment:
I thought it makes sense to pull these out as they aren't really rendering options.